Kitodo/src/main/java/org/kitodo/config/DigitalCollection.java

```diff
@@ -57,17 +57,17 @@ private DigitalCollections() {
         if (possibleDigitalCollection == null) {
             possibleDigitalCollection = new ArrayList<>();
         }
-        return DigitalCollections.possibleDigitalCollection;
+        return DigitalCollection.possibleDigitalCollection;
     }
 
     /**
      * Prepare digital collections and possible digital collections for process.
      *
-     * @param process
+     * @param projectBean
      *            object
      */
     @SuppressWarnings("unchecked")
-    public static void possibleDigitalCollectionsForProcess(Process process) throws JDOMException, IOException {
+    public static void possibleDigitalCollectionsForProcess(Project projectBean) throws JDOMException, IOException {
```

Comment:
is this still correct?

Reply:
Yes, as this thing only uses project title. So better is to call it with the project parameter instead of process.